### ISSUE TITLE
Bug with variable scoping

### ISSUE THREAD

**Issue by Tiliavir** [gulp-community/gulp-pug#151]
There seems to be a major issue with the scoping of variables, that are not available for all files.

Unfortunately I'm not sure which component of pug is the root cause for this.

I have prepared a small sample here:
https://github.com/Tiliavir/pug-variable-scope-issue

Simply CO, run `npm run build` and check the output vs. the resulting HTML. **Subtitle should be empty in most files, but are used in files where subtitle is not specified.**

My output is:

```
PS C:\pug-variable-scope-issue> npm run build

> var-issue@1.0.0 build C:\pug-variable-scope-issue
> node -v && npm -v && gulp

v6.9.1
3.10.8
[10:16:53] Using gulpfile C:\pug-variable-scope-issue\
[10:16:53] Starting 'default'...
[10:16:53] Finished 'default' after 17 ms
1 - undefined
2 - sub2
3 - undefined
4 - undefined
5 - sub5
6 - undefined
```

**Comment by Tiliavir** [comment]
Seems to be fixed with https://github.com/pugjs/gulp-pug/pull/146 are there any chances, that this is merged soon?

**Comment by jamen** [comment]
@Tiliavir Hey.  Yeah, I can merge these PRs and make a new release.

**Comment by Tiliavir** [comment]
I would really appreciate to see this fixed - is it tested for any other impacts? I've not reviewed the code to be honest ...

**Comment by jamen** [comment]
@Tiliavir I published `3.2.0` with the changes.  Let me know if that fixes.  They were backwards compatible.

**Comment by Tiliavir** [comment]
Fixed! Works like a charm ...
